### PR TITLE
Remove phasebanner, fix double pagemains

### PR DESCRIPTION
**Pull request by DamienBurkeWN**
### Setup

Requires https://github.com/FutureNorthants/northants-design-system/pull/109
Pull that in via yalc

### Testing
1. View home page and expect no difference
2. View a service landing page and expect no difference other than not having two <main> elements. Try each variation - with hero image, without, with top services, without, with hero image and top services, with alert banner, etc.
3. View other page types - news article, service page and expect no difference

### CODE REVIEW

File: frontend/src/cmsPages/ArticlePage/index.tsx [hunk 1]
```diff
@@ -9,7 +9,6 @@ import {
   Button,
   PageMain,
   Breadcrumbs,
-  PhaseBanner,
```
Comment (DamienBurkeWN) [inline frontend/src/cmsPages/ArticlePage/index.tsx:12]: Removing the phase banner is a P4 item in the backlog anyway and it simplified this work so out ya go...

File: frontend/src/cmsPages/ServiceLandingPage/index.tsx [hunk 2]
```diff
@@ -81,32 +80,27 @@ export default function ServiceLandingPage({
       {heroImage ? (
         <>
           <HeroImage {...heroImage} />
-          {serviceAlert?.title && (
```
Comment (DamienBurkeWN) [inline frontend/src/cmsPages/ServiceLandingPage/index.tsx:84]: Bonus bugfix, these alerts have _optional_ titles and if you didn't set one, you'd never see the alert at all!

File: frontend/src/cmsPages/ServiceLandingPage/index.tsx [hunk 3]
```diff
@@ -81,32 +80,27 @@ export default function ServiceLandingPage({
...
+          { serviceAlert?.alertType && (
             <MaxWidthContainer noPadding>
-              <PageMain>
-                <AlertBannerService {...serviceAlert}>
-                  {serviceAlert.children && <TextWithSlices html={serviceAlert.children.toString()} embeds={[]} />}
-                </AlertBannerService>
-              </PageMain>
+              <AlertBannerServiceIE />
```
Comment (DamienBurkeWN) [inline frontend/src/cmsPages/ServiceLandingPage/index.tsx:85]: Inadvertently removed in previous work

File: frontend/src/cmsPages/ServiceLandingPage/index.tsx [hunk 4]
```diff
@@ -81,32 +80,27 @@ export default function ServiceLandingPage({
       {heroImage ? (
         <>
           <HeroImage {...heroImage} />
-          {serviceAlert?.title && (
+          { serviceAlert?.alertType && (
             <MaxWidthContainer noPadding>
-              <PageMain>
```
Comment (DamienBurkeWN) [inline frontend/src/cmsPages/ServiceLandingPage/index.tsx:86]: The double main fix

File: frontend/src/components/AlertBannerService/index.tsx [hunk 5]
```diff
@@ -17,10 +17,10 @@ const AlertBannerServiceIE = (): ReactElement | null => {
         <p>
           It looks like you are using an out of date version of Internet Explorer. Using this browser will mean that
           certain features of this website will not work. It also means you are vulnerable to security exploits.{' '}
-          <strong>Microsoft will no longer support Internet Explorer from August 17th 2021.</strong>
+          <strong>The Internet Explorer 11 desktop application will go out of support for certain operating systems starting June 15, 2022.</strong>
```
Comment (DamienBurkeWN) [inline frontend/src/components/AlertBannerService/index.tsx:20]: Bonus correction

**Review by chrisrhymes — APPROVED** [review]
Tested with the changes to AlertServiceBanner in the design system and all good now.

**Pull request merged**